test/contrib/hdfs/webhdfs_client_test.py

```diff
@@ -16,15 +16,18 @@
 #
 import unittest
 
-from hdfs import InsecureClient
-from hdfs.ext.kerberos import KerberosClient
-from nose.plugins.attrib import attr
+import pytest
+try:
+    from hdfs import InsecureClient
+    from hdfs.ext.kerberos import KerberosClient
+except ImportError:
+    raise unittest.SkipTest('Unable to load hdfs module')
```

Comment:
You could probably also use [pytest.importorskip](https://docs.pytest.org/en/stable/reference.html#pytest-importorskip)?

Reply:
Thank you for your suggestion! fixed in d4de5fe